DigitalOffice/Services/UserService/RestRequests/UserCreateRequest.cs

```diff
@@ -8,5 +8,6 @@ public class UserCreateRequest
         public string Email { get; set; }
         public string Status { get; set; }
         public string Password { get; set; }
+        public bool IsAdmin { get; set; }
     }
```

Comment:
I think it's not needed here. 
We program admins by code, not requests. So it's appropriate that requests don't have such a property.
Thoughts?

Reply:
Yep, I thought like that too, but in our previous discussion Spartak said it's needed to be there.
Maybe it'll be false by default, but still be in request. And if admin wants to add other admin, this field will be "opened" for changes. So that it will be changed, oc.

Reply:
I see. Approved then!